packages/smplayer.rb

```diff
@@ -0,0 +1,64 @@
+require 'package'
+
+class Smplayer < Package
+  description 'SMPlayer is a free media player for Windows and Linux with built-in codecs that can play virtually all video and audio formats.'
+  homepage 'https://www.smplayer.info/'
+  version '24.5.0'
+  license 'GPL-2'
+  compatibility 'x86_64'
+  source_url "https://github.com/smplayer-dev/smplayer/releases/download/v#{version}/SMPlayer-#{version}-x86_64.AppImage"
+  source_sha256 '85fb5a2322f48a298b7784fc3516e672525593c017ca504310bff05e1330457b'
+
+  no_compile_needed
+
+  depends_on 'gtk3'
+  depends_on 'gdk_base'
+  depends_on 'sommelier'
+
+  def self.build
```

Comment:
Probably move this to the install section if you're setting `no_compile_needed`.

Reply:
This technically is a build step.

Reply:
Sorry, I misunderstood the code-- `no_compile_needed` skips building altogether:
![image](https://github.com/chromebrew/chromebrew/assets/64618338/b66d5eda-cf5c-49ea-ab4d-0308e55a7185)

Reply:
Right, build means to compile source into binaries.